### PR TITLE
Modified material in mpgd_barrel.xml

### PR DESCRIPTION
**Pull request by mposik1983**
Modified material in the inner and outer barrel layers, since exact material budget is not setup yet.
Overall material budget for inner MPGD barrel is now ~0.5% X0 and ~1% X0 for outer barrel layer.

### CODE REVIEW

File: compact/tracker/mpgd_barrel.xml [hunk 1]
```diff
@@ -49,6 +49,12 @@
     <constant name="MMDriftCuElectrode_thickness"           value="5*um"/>
     <constant name="MMDriftKapton_thickness"                value="250*um"/>
     <constant name="MMDriftCuGround_thickness"              value="0.41*um"/>
+    
+    <comment> FIXME: No support material is here, so fudge factor used to bring material budget to ~0.5% for barrel </comment>
+    <constant name="MMFudgeInnerMPGDBarrel_thickness"                      value="570*um"/>
+
+    <comment> FIXME: No support material is here, so fudge factor used to bring material budget to ~1% for barrel </comment>
+    <constant name="MMFudgeOuterMPGDBarrel_thickness"                      value="2.50*mm"/>
```
Comment (wdconinc) [inline compact/tracker/mpgd_barrel.xml:57]: Can you file a new issue corresponding with the FIXMEs here?

**Review by wdconinc — APPROVED** [review]

**Review by wdconinc — CHANGES_REQUESTED** [review]
There are still overlaps flagged by the geant4-overlap-check.

File: compact/tracker/mpgd_barrel.xml (hunk 1 above)
Comment (mposik1983) [reply compact/tracker/mpgd_barrel.xml:57]: I will file this issue soon, along with a couple others

**Review by wdconinc — APPROVED** [review]
Thanks for fixing the overlaps. The remaining failing check is because this is submitted with secrets only available to branches of the original repo. Happy to merge without those checks at this point.

**Comment by mposik1983** [comment]
> 

No problem. Do I close this PR now?

**Comment by wdconinc** [comment]
> No problem. Do I close this PR now?

No, when you close a PR before merging it doesn't go into the target branch.

**Pull request merged**